src/signified/__init__.py

```diff
@@ -1682,7 +1676,7 @@ def as_signal(val: HasValue[T]) -> Signal[T]:
     * [`unref`][signified.unref]: Function to dereference values.
 """
 
-HasValue: TypeAlias = Union[T, Computed[T], Signal[T]]
+HasValue: TypeAlias = Union[Computed[T], Signal[T], Flattener[T]]
```

Comment:
Not sure how I feel about this being hinted as `Flattener[T]` instead of just `T`.